### DISCUSSION TITLE
How to move agent from one Client Site to another?

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#1716] [Q&A] [answered]
Is it possible to move an agent installation from one Client/Site to another Client/Site without having to uninstall and reinstall the client?

This is something I will run across with my clients if I decide to implement this in our business.

**Comment by wh1te909** [comment, marked as answer]
yes, simply right click on the agent > Edit Agent. then select the new client/site from the dropdown and save.

**Reply by modem-19** [reply]
@wh1te909 Thanks!  I hadn't fully dug into editing the agent to see that.  I was expecting it to be more of a top level menu item.  But glad that was an easy answer!
